test/enigma_test.rb

```diff
@@ -77,6 +77,10 @@ def test_it_can_generate_keys
     assert_equal "00006", @enigma.generate_key
   end
 
+  def test_user_input_is_an_empty_array_by_default
+    assert_equal [], @enigma.user_input
+  end
```

Comment:
Confirms that it is an empty array by default, not able to test other values when arguments are given. I stub this method in all other tests where it comes up, but this test allows for slightly more robust testing as it at least covers the default return value.